src/modules/task.js

```diff
@@ -0,0 +1,35 @@
+// Declare tasks array
+const tasks = [];
+
+// Create class declaration for tasks in task array
+class Task {
+  constructor(description, completed = false) {
+    this.index = tasks.length + 1;
+    this.description = description;
+    this.completed = completed;
+  }
+
+  addTask() {
+    tasks.push(this);
+  }
+
+  removeTask() {
+    const index = tasks.indexOf(this);
+    tasks.splice(index, 1);
+    for (let i = index; i < tasks.length; i += 1) {
+      tasks[i].index -= 1;
+    }
+  }
```

Comment:
It'd be fine if you switch your methods syntax to ES6. Even though your current code will work, the ES6 syntax is the latest that's recommended to be used.